libnetdata/dictionary/dictionary.h

```diff
@@ -13,7 +13,7 @@ struct dictionary_stats {
 };
 
 typedef struct name_value {
-    avl avl;                // the index - this has to be first!
+    avl avl_instance;       // the index - this has to be first!
```

Comment:
```suggestion
    avl avl_node;           // the index - this has to be first!
```
Let's keep it consistent with the similar change above.